test/Ocelot.UnitTests/Configuration/DefaultMetadataCreatorTests.cs

```diff
@@ -0,0 +1,97 @@
+﻿using Ocelot.Configuration;
+using Ocelot.Configuration.Creator;
+using Ocelot.Configuration.File;
+
+namespace Ocelot.UnitTests.Configuration;
+
+public class DefaultMetadataCreatorTests : UnitTest
+{
+    private FileGlobalConfiguration _globalConfiguration;
+    private Dictionary<string, string> _metadataInRoute;
+    private MetadataOptions _result;
+    private readonly DefaultMetadataCreator _sut = new();
+
+    [Fact]
+    public void Should_return_empty_metadata()
+    {
+        this.Given(_ => GivenEmptyMetadataInGlobalConfiguration())
+            .Given(_ => GivenEmptyMetadataInRoute())
+            .When(_ => WhenICreate())
+            .Then(_ => ThenDownstreamRouteMetadataMustBeEmpty());
```

Comment:
We no longer use BDDfy for unit testing. Replace BDDfy expressions with the AAA pattern, and use xUnit and Shouldly instead.

Reply:
No sorry, I don't have time for that.

Reply:
Typically, I don't have much time. However, I make an effort to set aside BDDfy when writing new tests and creating new unit testing classes. Have you ever examined the call stack while debugging unit tests which written with BDDfy? It's quite a hassle!

Reply:
Well. Let's say next time.